pkg/skaffold/deploy/kpt/kpt_test.go

```diff
@@ -115,9 +115,10 @@ func TestKpt_Deploy(t *testing.T) {
 				- foo.yaml`},
 			commands: testutil.
 				CmdRunOut("kpt fn source .", ``).
+				AndRunOut("kpt fn source kpt-func.yaml", ``).
 				AndRunOut(fmt.Sprintf("kpt fn sink %v", tmpKustomizeDir), ``).
 				AndRunOut(fmt.Sprintf("kustomize build %v", tmpKustomizeDir), ``).
-				AndRunOut("kpt fn run --dry-run --fn-path kpt-func.yaml", testPod).
+				AndRunOut("kpt fn run --dry-run", testPod).
```

Comment:
so, it ok to run `kpt fn run` ignoring the user defined `kpt-func.yaml` ?

Reply:
haha, it's a wrong usage in previous code. fn-path refers to directory, not a single file.